### ISSUE TITLE
How can run this in background

### ISSUE THREAD

**Issue by johnfelipe** [WhiteCoreSim/WhiteCore-Dev#260]
Hi bro, me again, in opensim robust grid instance im using this commands for run in background

>screen -S Grid mono Robust.exe

and 

>screen -S Zone01 mono OpenSim.exe

How can do that in WCS?

**Comment by emperorstarfinder** [comment]
The same would apply for WhiteCore-Sim as well.  in Linux screen -S refers to the screen program.  

screen -S Grid mono WhiteCore.Server.sh

screen -S Region mono WhiteCore.sh

Windows does not have the screen program and thus this would be an entirely different process.

**Comment by johnfelipe** [comment]
For linux do u have other method not using screen because sometimes that screen terminated i dont know why...

**Comment by emperorstarfinder** [comment]
Unfortunately screen is the only way for Linux operating systems as far as I know.  There may be another program similar to screen for Linux but I am not personally aware of it.  Usually when the consoles terminate from screen however they will still actually be running but rebooting the machine is the only way to actually get them back into screen.

**Comment by johnfelipe** [comment]
This not work:

screen -S Grid mono WhiteCore.Server.sh

screen -S Region mono WhiteCore.sh

and not work to:

	cd WhiteCore-Dev/

	cd WhiteCoreSim/bin/

	screen -S Grid
	./WhiteCore.Server.sh

and show me this:

	felipe@gtic:~/WhiteCore-Dev/WhiteCoreSim/bin$ ./WhiteCore.Server.sh
	Warning: Degraded allocation.  Consider increasing nursery-size if the warning persists.
	Warning: Degraded allocation.  Consider increasing nursery-size if the warning persists.
	Warning: Repeated degraded allocation.  Consider increasing nursery-size.
	Stack overflow: IP: 0x7f518d831b95, fault addr: 0x7ffd3dd25f58
	Stacktrace:
	  at <unknown> <0xffffffff>

**Comment by emperorstarfinder** [comment]
Good Morning,

That looks like an issue with Mono itself.  the latest mono has a bug relating to lower memory usage.  I did find a way to fix memory on mono itself which I will post once I fully wake up.

**Comment by greythane** [comment]
I need to get up to speed on all the conversations but...
In the WhiteCoreSim install top directory there should be a script that invokes the use of screen for both the grid and region server, namely... run_gridmode.sh
A similar script exists, run_standalone.sh, for the standalone setup.

As far as your error messages these are indeed mono related.
Can you let me know your configuration?  OS, version, x86/x64, mono version etc.
I have tested WhiteCore in quite a few environments and not seen this particular error.
G.

**Comment by emperorstarfinder** [comment]
Good morning,  

I have the script prepped that will fix the problem.  Its not anything caused by WhiteCore itself but rather a direct bug in mono.  This bug had been introduced by Mono a few years back and had been fixed and apparently is reintroduced.  I have found people mentioning it in Ubuntu forums and other forums related to Linux. I will upload the script this morning and post a link to it along with instructions to get it running in background.
